### PR TITLE
Hw25 thursday

### PR DESCRIPTION
**Pull request by fyarkiy**

### CODE REVIEW

**Review by Victoria-Moiseienko — COMMENTED** [review]
Good job :)

File: src/main/java/internet/store/controllers/ordercontroller/OrderByUserController.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+import internet.store.lib.Injector;
+import internet.store.service.OrderService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class OrderByUserController extends HttpServlet {
```
Comment (dmytro-hryhoruk) [inline src/main/java/internet/store/controllers/ordercontroller/OrderByUserController.java]:
```suggestion
public class GetUserOrdersController extends HttpServlet {
```

File: src/main/java/internet/store/controllers/ordercontroller/DisplayAllOrderController.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+import internet.store.lib.Injector;
+import internet.store.service.OrderService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class DisplayAllOrderController extends HttpServlet {
```
Comment (dmytro-hryhoruk) [inline src/main/java/internet/store/controllers/ordercontroller/DisplayAllOrderController.java]:
```suggestion
public class GetAllOrdersController extends HttpServlet {
```

File: src/main/webapp/WEB-INF/views/product/manage.jsp [hunk 3]
```diff
@@ -0,0 +1,47 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%--
+  Created by IntelliJ IDEA.
+  User: Admin
+  Date: 11.09.2020
+  Time: 17:39
+  To change this template use File | Settings | File Templates.
+--%>
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/WEB-INF/views/product/manage.jsp]: remove comments

File: src/main/java/internet/store/controllers/productcontroller/ManageProductController.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+import internet.store.lib.Injector;
+import internet.store.service.ProductService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class ManageProductController extends HttpServlet {
```
Comment (dmytro-hryhoruk) [inline src/main/java/internet/store/controllers/productcontroller/ManageProductController.java]:
```suggestion
public class ManageProductsController extends HttpServlet {
```

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**